libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java

```diff
@@ -1011,7 +1011,13 @@ public void run() {
             Thread.currentThread().interrupt();
         }
 
-        return StringUtils.notNullStr(mContentHtml);
+        mContentHtml = StringUtils.notNullStr(mContentHtml);
+        // Remove ZERO WIDTH SPACE character - http://www.fileformat.info/info/unicode/char/200b/index.htm
+        // from the beginning of the content. See: https://github.com/wordpress-mobile/WordPress-Android/issues/5009
+        if (mContentHtml.charAt(0) == '\u200B') {
```

Comment:
This causes a `StringIndexOutOfBoundsException` if you switch to HTML mode (or exit the editor, or wait for autosave) before adding any content. 

```
Process: org.wordpress.android.beta, PID: 22501
java.lang.StringIndexOutOfBoundsException: length=0; index=0
  at java.lang.String.charAt(Native Method)
  at org.wordpress.android.editor.EditorFragment.getContent(EditorFragment.java:1017)
  at org.wordpress.android.editor.EditorFragment$6.run(EditorFragment.java:650)
```